chapter2/tests/t8/chapter2-l4-t8-get-tcp-listening-connections.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Task 8 - display new tcp listening connections in the range 10000-10100
+set -eo pipefail
+
+
+# set the port range
+readonly PORT_RANGE_START='10000'
+readonly PORT_RANGE_END='10100'
+
+
+# port range expr
+port_range="sport >= :$PORT_RANGE_START and sport <= :$PORT_RANGE_END"
+
+
+# Check if there are any TCP connections listening on the specified port range
+while :
+do
+  if [[ -z $(ss -tano "${port_range}" | grep -v "State") ]]; then
+    # If there are no listening connections, print a message and wait for 2 seconds
+    echo 'No TCP listening sockets found bound to ports in 10000-10100 range'
```

Comment:
> Why you have in some places single quotes and in other double ones?

@kseniyashaydurova 

Just because..
In general to refuse single?
![123](https://user-images.githubusercontent.com/93929010/233624752-ac1c8302-344c-41ee-bbd9-29412b754914.png)

Reply:
I just suggest you to stick to the same rules related to quotes, i.e. use double quotes everywhere in a file (except places like this `awk '{printf "%-20s %-20s\n", $4, $5}'`), where you improve readability by adding another type of quotes. Now it looks like a mess when you have one kind of quotes in one place and another in others. We always try to follow the same quoting concept in our scripts

Reply:
> I just suggest you to stick to the same rules related to quotes, i.e. use double quotes everywhere in a file (except places like this `awk '{printf "%-20s %-20s\n", $4, $5}'`), where you improve readability by adding another type of quotes. Now it looks like a mess when you have one kind of quotes in one place and another in others. We always try to follow the same quoting concept in our scripts


@kseniyashaydurova 
Ok, done, i'll do this way